tests/unit_tests/scheduler/test_scheduler.py

```diff
@@ -20,7 +20,7 @@
 )
 from ert.run_arg import RunArg
 from ert.scheduler import LsfDriver, OpenPBSDriver, create_driver, scheduler
-from ert.scheduler.job import State
+from ert.scheduler.job import State as JobState
```

Comment:
Would it make sens to rename `ert.scheduler.job.State` to `ert.scheduler.job.JobState`?